src/index.ts

```diff
@@ -4,6 +4,12 @@ import * as TrackerLayer from "./protocol/tracker_layer/index"
 import * as Errors from "./errors/index"
 import * as Utils from "./utils/index"
 
+export * from "./protocol/control_layer/index"
+export * from "./protocol/message_layer/index"
+export * from "./protocol/tracker_layer/index"
+export * from "./errors/index"
+export * from "./utils/index"
```

Comment:
Are the classes in `utils-directory` mostly internal helper utilities? Maybe we don't need to export those.

Reply:
Good thought but no, many of these aren't used internally at all.

Utils currently contains:
```js
[
  'TimestampUtil',
  'OrderingUtil',
  'StreamMessageValidator',
  'CachingStreamMessageValidator',
  'SigningUtil',
  'TrackerRegistry',
  'createTrackerRegistry',
  'getTrackerRegistryFromContract'
]
```

Reply:
Also note any internal stuff in `utils` is already not exported e.g. there's no export for `utils/OrderedMsgChain.ts`.

This PR only re-exports stuff that was already exported.